test/runtests.jl

```diff
@@ -0,0 +1,20 @@
+using ChainRulesTestUtils
+using Distributions
```

Comment:
Is this used anywhere?

Reply:
Yeah for `10rand()` in the test below. I wasn't sure which tests to add in so I just randomly picked some up from `ChainRules`.

Reply:
`rand` is defined in `Base`